jib-core/src/integration-test/java/com/google/cloud/tools/jib/api/ContainerizerIntegrationTest.java

```diff
@@ -271,9 +276,9 @@ public void testSteps_forBuildToDockerRegistry_skipExistingDigest()
   public void testBuildToDockerRegistry_dockerHubBaseImage()
       throws InvalidImageReferenceException, IOException, InterruptedException, ExecutionException,
           RegistryException, CacheDirectoryCreationException {
-    buildRegistryImage(
-        ImageReference.parse("openjdk:8-jre-alpine"),
-        ImageReference.of("localhost:5000", "testimage", "testtag"),
+    buildImage(
+        ImageReference.parse("openjdk:8-jre-slim"),
```

Comment:
Nice, I'm guessing this fixes the compatibility issue and also results in a smaller image:)